neurogym/envs/annubes.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+
+import neurogym as ngym
+from neurogym import TrialEnv
+
+
+class AnnubesEnv(TrialEnv):
+    def __init__(
+        self,
+        session=None,
+        stim_intensities=None,
+        stim_time=1000,
+        decision_time=100,
+        catch_prob=0.5,
+        fix_intensity=0,
+        fix_time=500,
+        dt=100,
+        tau=100,
+        n_outputs=2,
+        output_behavior=None,
+        noise_std=0.01,
+        random_seed=None,
+    ):
+        if output_behavior is None:
+            output_behavior = [0, 1]
+        if stim_intensities is None:
+            stim_intensities = [0.8, 0.9, 1.0]
+        if session is None:
+            session = {"v": 0.5, "a": 0.5}
+        super().__init__(dt=dt)
+
+        self.session = session
+        self.stim_intensities = stim_intensities
+        self.stim_time = stim_time
+        self.decision_time = decision_time
+        self.catch_prob = catch_prob
+        self.fix_intensity = fix_intensity
+        self.fix_time = fix_time
+        self.dt = dt
+        self.tau = tau
+        self.n_outputs = n_outputs
+        self.output_behavior = output_behavior
+        self.noise_std = noise_std
+        self.random_seed = random_seed
+        alpha = dt / self.tau
+        self.noise_factor = self.noise_std * np.sqrt(2 * alpha) / alpha
+        # Set random state
+        if random_seed is None:
+            rng = np.random.default_rng(random_seed)
+            random_seed = rng.integers(2**32)
+        self._rng = np.random.default_rng(random_seed)
+        self._random_seed = random_seed
+        self.timing = {"fixation": self.fix_time, "stimulus": self.stim_time, "decision": self.decision_time}
+        # Set the name of each input dimension
+        obs_space_name = {"fixation": 0, "start": 1, "v": 2, "a": 3}
```

Comment:
Should this be hard coded? In theory someone could set up `sessions` with `t` for tactile (or whatever else), and then I think this would lead to a problem.

Would this work?

```suggestion
        obs_space_name = {"fixation": 0, "start": 1}
        for i, trial in enumerate(session, 2):
            obs_space_name[trial: i]
```

Reply:
Also, does catch need to be included here as well?

Reply:
Good point, I edited `obs_space_name` accordingly. Catch does not need to be included here.